config/routes.rb

```diff
@@ -672,7 +672,7 @@ def call(env)
         end
 
         scope module: :services do # this api has a knack for inconsistency
-          resources :backend_apis, controller: :backend_api_configs, except: %i[new edit], defaults: { format: :json }
+          resources :backend_api_configs, path: 'backend_usage', except: %i[new edit], defaults: { format: :json }
```

Comment:
This should be plural

Reply:
Ok, let's keep the conversation in 1 place: https://github.com/3scale/porta/pull/1337#discussion_r336521107